### ISSUE TITLE
role prop not supported unlike accessibilityRole

### ISSUE THREAD

**Issue by getusha** [FormidableLabs/eslint-plugin-react-native-a11y#151]
Encountering the following error:

```
Missing a11y props. Expected one of: accessibilityRole OR BOTH accessibilityLabel + accessibilityHint OR BOTH accessibilityActions + onAccessibilityAction
```
Despite having a valid `role` value, this error is being triggered.

Example
```
<Image role="img" />
```

accessibilityRole is deprecated in react-native-web additionally the `role` prop has precedence over the accessibilityRole prop.

**Comment by SpadarShut** [comment]
Any plans to merge this PR?
